services/install-mariadb.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/mariadb
+
+  # create yml(s)
+  {
+    echo "version: '3'"
+    echo "services:"
+    echo "  mariadb:"
+    echo "    image: jsurf/rpi-mariadb"
+    echo "    container_name: rpi-mariadb"
+    echo "    ports:"
+    echo "      - 3306:3306"
+    echo "    environment:"
+    echo "      - MYSQL_ROOT_PASSWORD=my-secret-pw"
```

Comment:
@dogi 

We can replace `    echo "      - MYSQL_ROOT_PASSWORD=my-secret-pw"` with `    echo "      - MYSQL_ROOT_PASSWORD=${password}"` and have a `.env` file with `password=`.

https://docs.docker.com/compose/env-file/